src/mscorlib/src/System/Environment.cs

```diff
@@ -1752,7 +1751,6 @@ public enum SpecialFolder {
             //      GetWindowsDirectory()
             //
             Windows                = Win32Native.CSIDL_WINDOWS,
-#endif // !FEATURE_CORECLR
```

Comment:
Per my comments earlier about Environment, these changes shouldn't be necessary.